styles/generics/_mixins-icons.scss

```diff
@@ -2,60 +2,294 @@
 $MIXINS ICONS
 \*------------------------------------*/
 
-@mixin iconfont {
-	position: relative;
-	font-family: 'Icons';
-	speak: none;
-	font-style: normal;
-	font-weight: normal;
-	font-variant: normal;
-	text-transform: none;
-	line-height: 0;
-	left: 0;
-	color: $soft-script;
-
-	/* Better Font Rendering =========== */
-  @include font-rendering;
+@mixin iconfont($icon: "") {
+  &:before {
+    @include font-rendering;
+  	color: $soft-script;
+  	content: $icon;
+  	font-family: 'Icons';
+  	font-style: normal;
+  	font-variant: normal;
+  	font-weight: normal;
+    left: 0;
```

Comment:
- Missing tab
[OPTIONAL]: If you like...     

font: {
        size: ...;
        weight: ...;
    }